### DISCUSSION TITLE
Disable the new search bar in Windows 11 start menu/search home

### DISCUSSION THREAD

**Discussion by XuanChen1234** [valinet/ExplorerPatcher#1713] [Ideas]
![LXUfR5q9PckFqqHAoP9gdj](https://github.com/valinet/ExplorerPatcher/assets/140480110/ebb61619-127d-4a01-9298-fcb648b7a496)

**Comment by milestprower92** [comment]
> ![LXUfR5q9PckFqqHAoP9gdj](https://private-user-images.githubusercontent.com/140480110/256986226-ebb61619-127d-4a01-9298-fcb648b7a496.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzMTg2NjcsIm5iZiI6MTcwMjMxODM2NywicGF0aCI6Ii8xNDA0ODAxMTAvMjU2OTg2MjI2LWViYjYxNjE5LTEyN2QtNGEwMS05Mjk4LWZjYjY0OGI3YTQ5Ni5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxMVQxODEyNDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yYjg5NzViZmRkNTI2YzE4ODUwNjc0Y2NhYTcxOGUxNDg4ZmRhN2E4YmFkMTMxMmE0NTE2ZmY4ODEwY2M4NDcwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Y8Dq-fF3pncwaY717Tbk90ZTc7VO7Sc1i-yYD5dmFKo)

not possible
